.github/workflows/build.yml

```diff
@@ -0,0 +1,187 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Build"
```

Comment:
The name used here does not need to match the name of the workflow definition file.

However, I think it makes sense when they have at least a similar name.